astropy/cosmology/parameters.py

```diff
@@ -63,7 +63,8 @@
 # LOCAL
 from .realizations import available
 
-__all__ = ["available"] + list(available)
+__all__ = ["available"]
+__all__ += list(available)
```

Comment:
This doesn't make sense. Why break a single line to two different lines? What would be the advantage of this?

Reply:
I really don't like this over at `io/fits/__init__.py`. Can we not do this one?

Reply:
I can ignore it, but we won't gain the future advantages.

Reply:
I'm closing this comment in favor of the one on ``io/fits/__init__.py``.